slang-kotlin/src/main/java/com/sonarsource/slang/kotlin/KotlinTreeVisitor.java

```diff
@@ -120,7 +120,7 @@ private Tree createElement(@Nullable PsiElement element) {
     TreeMetaData metaData = getTreeMetaData(element);
 
     if (element instanceof PsiErrorElement) {
-      throw new IllegalStateException("Cannot convert file due to syntactic errors");
+      throw new ParseException("Cannot convert file due to syntactic errors");
```

Comment:
Shouldn't we give the position to the ParseException? The ParseException constructor with a TextPointer is never used for the moment...